.circleci/config.yml

```diff
@@ -70,9 +70,18 @@ jobs:
             ./mconfig -p /usr/local
             make -j$(nproc) -C ./builddir
             sudo make -C ./builddir install
+      - run:
+          name: Install aws cli
+          command: |
+            pip install awscli --upgrade --user
+            aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID
+            aws configure set aws_secret_access_key $AWS_SECRET_ACCESS_KEY
+            aws configure set default.region $AWS_DEFAULT_REGION
```

Comment:
Technically you don't need to configure `AWS_ACCESS_KEY_ID`/`AWS_SECRET_ACCESS_KEY`/`AWS_DEFAULT_REGION ` explicitly, the `aws` CLI pulls those from the environment by default.

Reply:
Thanks! Removing